h2o-docs/src/product/data-science/algo-params/data_leakage_handling.rst

```diff
@@ -0,0 +1,31 @@
+``data_leakage_handling``
+-------------------------
+
+- Available in: Target Encoding
+
+Description
+~~~~~~~~~~~
+The ``data_leakage_handling`` parameter defines the strategy used to compute the target average in order to avoid data leakage. Overfitting can be prevented by removing some holdout data when calculating the target average on the training data.
+
+The following strategies can be specified:
+
+-  ``none``: no holdout. The mean is calculated on all rows of data \*\*. This should be used for validation or test data.
```

Comment:
Is there a reason for the "\*\*"? On the local build, it just appears as "The mean is calculated on all rows of data **."

Reply:
mmm, it's legacy. Was already in the old `holdout_type.rst`.
Let me remove it.